test/integration/01-setup/01-lightning-channels.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { once } from "events"
+import { find } from "lodash"
+import { MainBook } from "src/mongodb"
+import { lndFeePath } from "src/ledger/ledger"
+import {
+  checkIsBalanced,
+  closeChannel,
+  getChannels,
+  getWalletInfo,
+  lnd1,
+  lnd2,
+  lndOutside1,
+  lndOutside2,
+  mineBlockAndSync,
+  openChannelTesting,
+  setChannelFees,
+  subscribeToChannels,
+  subscribeToGraph,
+  updateRoutingFees,
+  waitFor,
+  waitUntilSync,
+} from "test/helpers"
+import { offchainLnds, onChannelUpdated, updateEscrows } from "src/lndUtils"
+import { sleep } from "src/utils"
+
+jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
+
+//this is the fixed opening and closing channel fee on devnet
+const channelFee = 7637
+const lnds = [lnd1, lnd2, lndOutside1, lndOutside1]
+let channelLengthMain, channelLengthOutside1
+
+beforeEach(async () => {
+  await waitUntilSync({ lnds })
+  channelLengthMain = (await getChannels({ lnd: lnd1 })).channels.length
+  channelLengthOutside1 = (await getChannels({ lnd: lndOutside1 })).channels.length
+})
+
+afterEach(async () => {
+  await checkIsBalanced()
+})
+
+// Setup the next network
+// lnd2 <- lnd1 <-> lndOutside1
+// lnd2 <- lnd1 -> lndOutside2
+// this setup avoids close channels for routing fees tests
+describe("Lightning channels", () => {
```

Comment:
don't you think this test could be rewritten?
we could just have an object defining the channel we want to have:

```
[{
initiator: lnd1,
partnre: lnd2,
basefee: 1,
...
},{
initiator: lnd1,
partnre: lnd2,
basefee: 1,
}]
```